recipe/meta.yaml

```diff
@@ -13,19 +13,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1004
+  number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   host:
     - python
     - setuptools
-    - argparse    # [py26]
```

Comment:
Dropping Python 2isms as it is no longer supported

xref: https://github.com/jayclassless/setoptconf/pull/3